sample-applications/blazing-coffee/BlazingCoffee/Client/wwwroot/js/themeChooser.js

```diff
@@ -3,45 +3,27 @@
         return window.localStorage['ThemeSetting'];
     },
     setTheme: function (themeName) {
-        if (themeName === "auto") {
-            window.localStorage.removeItem('ThemeSetting');
-            this.autoDetectTheme();
-        } else {
+        if (themeName) {
             window.localStorage['ThemeSetting'] = themeName;
             this.changeTheme(themeName);
         }
     },
     changeTheme: function (themeName) {
         // Build the new css link         
+        let cdnBase = `https://blazor.cdn.telerik.com/blazor/3.4.0/kendo-theme-${themeName || 'default'}/all.css`;
```

Comment:
The version is hardcoded - should be extracted to some config? At least, we might want to stick to latest `3.5.0` instead of `3.4.0`.